CHANGELOG.md

```diff
@@ -28,6 +30,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### ⚠️ Deprecated
 
+## [0.6.0] - 2024-09-16
+
+### 🎉 Added
+
+- [#67](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/67) Adds optimization_preferences as an option for TranspilerService
+- [#70](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/70) Enabling coupling_map as options to the AI Synthesis passes
+
+### ✏️ Changed
+
+- [#63](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/63) Fixing the type of ECR gates when loading circuits from the qiskit transpiler service
+- [#52](https://github.com/Qiskit/qiskit-ibm-transpiler/pull/52) Keep trying getting result even in case of HTTPError exception with status 520
```

Comment:
These should probably be in the release notes file 0.6.0.rst? They're bug fixes? 

You can manually add them like this:

https://github.com/Qiskit/qiskit-ibm-transpiler/blob/8c8168eff1b780cef9ead93a223e4f5069c6d459/release-notes/0.4.9.rst?plain=1#L4-L7

Reply:
Sorry! I missed this last comment.
Here is the fix: https://github.com/Qiskit/qiskit-ibm-transpiler/pull/72